src/env.js

```diff
@@ -13,6 +13,6 @@
     window.__env.cacheServerUrl = 'https://identitycache-dev.energyweb.org/v1';
     window.__env.natsServerUrl = 'https://identityevents-dev.energyweb.org/';
     window.__env.kmsServerUrl = undefined;
-    window.__env.stakingPoolFactoryAddress = '0x3490B3e5E7C1f696c61FDe1c2679725B19274B0e';
+    window.__env.stakingPoolFactoryAddress = '0xd61Ac65c0fECfb83108f6E374a57FfF9c692901d';
```

Comment:
Shouldn`t this stuff be taken from a .env file and not commited? Only if it`s sensitive data. If not, they maybe it`s ok

Reply:
It should, but right now devops do not take care of it. So it is maintained by repo.